text/chapter6.md

```diff
@@ -8,6 +8,8 @@ This motivating example for this chapter will be a library for hashing data stru
 
 We will also see a collection of standard type classes from PureScript's Prelude and standard libraries. PureScript code leans heavily on the power of type classes to express ideas concisely, so it will be beneficial to familiarize yourself with these classes.
 
+If you come from an Object Oriented background, please note that the word "class" means something _very_ different in this context than what you're used to. A type class serves a purpose more similar to an OO interface. 
```

Comment:
Feel free to nix my extra commentary.  It seemed to me that immediately dispelling any preconceived notions about the word "class" might be helpful